docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -256,13 +256,8 @@ You can disable column virtualization by setting the column buffer to a higher n
 
 ## Components prop
 
-As part of the customization API, the grid allows you to replace and override nested components with the `components` prop. The prop accepts an object with the following keys:
-
-- `toolbar`: The component rendered above the column header bar.
-- `loadingOverlay`: The component rendered when the loading react prop is set to true.
-- `noRowsOverlay`: The component rendered when the rows react prop is empty or [].
-- `footer`: The component rendered below the viewport.
-- `pagination`: The component rendered for the pagination feature.
+As part of the customization API, the grid allows you to replace and override nested components with the `components` prop.
+The prop accepts an object of type [GridSlotsComponent](/api/data-grid/#slots) .
```

Comment:
Why do you have this changes? You should just revert this file

Reply:
I pulled from master, that's what I got from it.

Reply:
These are not marked as changes in the diff